scripts/apps/dashboard/world-clock/timezones-all.json

```diff
@@ -1 +1,1979 @@
+{
+    "zones": {
+        "Africa/Abidjan": ["-0:16:8 - LMT 1912 -0:16:8", "0 - GMT"],
+        "Africa/Accra": ["-0:0:52 - LMT 1918 -0:0:52", "0 Ghana %s"],
+        "Africa/Cairo": ["2:5:9 - LMT 1900_9 2:5:9", "2 Egypt EE%sT"],
+        "Africa/Casablanca": [
+            "-0:30:20 - LMT 1913_9_26 -0:30:20",
+            "0 Morocco WE%sT 1984_2_16",
+            "1 - CET 1986 1",
+            "0 Morocco WE%sT"
+        ],
+        "Africa/Dakar": [
+            "-1:9:44 - LMT 1912 -1:9:44",
+            "-1 - WAT 1941_5 -1",
+            "0 - GMT"
+        ],
+        "Africa/Johannesburg": [
+            "1:52 - LMT 1892_1_8 1:52",
+            "1:30 - SAST 1903_2 1:30",
+            "2 SA SAST"
+        ],
+        "Africa/Khartoum": [
+            "2:10:8 - LMT 1931 2:10:8",
+            "2 Sudan CA%sT 2000_0_15_12 2",
+            "3 - EAT"
+        ],
+        "Africa/Kinshasa": ["1:1:12 - LMT 1897_10_9 1:1:12", "1 - WAT"],
+        "Africa/Lagos": ["0:13:36 - LMT 1919_8 0:13:36", "1 - WAT"],
+        "Africa/Nairobi": [
+            "2:27:16 - LMT 1928_6 2:27:16",
+            "3 - EAT 1930 3",
+            "2:30 - BEAT 1940 2:30",
+            "2:45 - BEAUT 1960 2:45",
+            "3 - EAT"
+        ],
+        "Africa/Tunis": [
+            "0:40:44 - LMT 1881_4_12 0:40:44",
+            "0:9:21 - PMT 1911_2_11 0:9:21",
+            "1 Tunisia CE%sT"
+        ],
+        "America/Anchorage": [
+            "14:0:24 - LMT 1867_9_18 14:0:24",
+            "-9:59:36 - LMT 1900_7_20_12 -9:59:36",
+            "-10 - CAT 1942 -10",
+            "-10 US CAT/CAWT 1945_7_14_23",
+            "-10 US CAT/CAPT 1946 -10",
+            "-10 - CAT 1967_3 -10",
+            "-10 - AHST 1969 -10",
+            "-10 US AH%sT 1983_9_30_2 -9",
+            "-9 US Y%sT 1983_10_30 -9",
+            "-9 US AK%sT"
+        ],
+        "America/Argentina/Buenos_Aires": [
+            "-3:53:48 - LMT 1894_9_31 -3:53:48",
+            "-4:16:48 - CMT 1920_4 -4:16:48",
+            "-4 - ART 1930_11 -4",
+            "-4 Arg AR%sT 1969_9_5 -4",
+            "-3 Arg AR%sT 1999_9_3 -3",
+            "-4 Arg AR%sT 2000_2_3 -3",
+            "-3 Arg AR%sT"
+        ],
+        "America/Bogota": [
+            "-4:56:16 - LMT 1884_2_13 -4:56:16",
+            "-4:56:16 - BMT 1914_10_23 -4:56:16",
+            "-5 CO CO%sT"
+        ],
+        "America/Caracas": [
+            "-4:27:44 - LMT 1890 -4:27:44",
+            "-4:27:40 - CMT 1912_1_12 -4:27:40",
+            "-4:30 - VET 1965 -4:30",
+            "-4 - VET 2007_11_9_03 -4",
+            "-4:30 - VET"
+        ],
+        "America/Chicago": [
+            "-5:50:36 - LMT 1883_10_18_12_9_24 -5:50:36",
+            "-6 US C%sT 1920 -6",
+            "-6 Chicago C%sT 1936_2_1_2 -6",
+            "-5 - EST 1936_10_15_2 -5",
+            "-6 Chicago C%sT 1942 -6",
+            "-6 US C%sT 1946 -6",
+            "-6 Chicago C%sT 1967 -6",
+            "-6 US C%sT"
+        ],
+        "America/Edmonton": [],
```

Comment:
does it actually work without any specs?

Reply:
it does, I tried it with specs, without them, and with wrong specs. I think they're just useful if the actual timezone id doesn't exist on moment.js, and this one does exist

Reply:
I see, cool